landlab/components/priority_flood_flow_router/priority_flood_flow_router.py

```diff
@@ -375,6 +353,13 @@ def __init__(
         keyword arguments, tests the argument of runoff, and
         initializes new fields.
         """
+        if (richdem := self.load_richdem()) is None:
+            raise RuntimeError(
+                "PriorityFloodFlowRouter requires richdem but richdem is not installed"
+            )
```

Comment:
Maybe we can help the user here by pointing out how to install? Not sure if we would recommend the conda version if this is not maintained though...

Reply:
It looks like you're looking at an old version. I've added a better error message but I guess it's not showing up. Maybe I forgot to push it to GitHub. 🤦

Reply:
@BCampforts I'm not sure what's going on exactly but I think you must have commented on an old commit. This is code is not present in the latest version.